VHAR-3336 Yhdenmukaista loppujenkin valintalistaitemien tyylit

jarnovayrynen opened the pull request.

jarnovayrynen committed:
VHAR-3336 Yhdenmukaista loppujenkin valintalistaitemien tyylit

jarnovayrynen committed:
Lopeta xml:ien lokittaminen TUR ja työtunnit -poikkeuksissa

jarnovayrynen committed:
Merge branch 'develop' into VHAR-3336

rckopteri reviewed (approved):
En testannut tätä omalla koneella, eli siis tarkkaa käsitystä, että mihin kaikkialle tämä vaikuttaa, ei ole, mutta hyvältä näyttää.
Tykkään kovasti, että saadaan tyylejä yhtenäisiksi.

jarnovayrynen committed:
Korjaa pudotusvalikon ulkopuolella klikkaamisen hanskaus

Poista turhat :klikattu-ulkopuolelle-params passaamiset
Korjaa ilmoitukset-näkymän React-varoitukset (avaimia puuttui)

jarnovayrynen committed:
Korjaa TypeError, chevron id:t tarkoituksella samat up ja down
